orga/app/components/campaign/activity/participants-list.hbs

```diff
@@ -1,52 +1,50 @@
-<div ...attributes>
+<section ...attributes>
   <div class="panel">
-    <div class="table content-text content-text--small">
-      <table>
-        <thead>
-          <tr>
-            <th>{{t 'pages.campaign-activity.table.column.last-name'}}</th>
-            <th>{{t 'pages.campaign-activity.table.column.first-name'}}</th>
+    <table class="table content-text content-text--small">
```

Comment:
[suggestion] (hors context)
Toutes nos tableaux ont le même style au niveau de la police.
Je propose de mettre les styles de `content-text` et `content-text--small` dans la classe `table` afin d'éviter de le répéter sur chaque tableau

Reply:
Est-ce que ce serait pas pertinent de créer un composant de tableau ? Dans Pix Orga dans un premier temps et why not dans Pix UI

Reply:
vu avec @bpetetot, on va remettre à plat la typo dans une autre PR